vulcan/lib/server/controllers/sessions_controller.rb

```diff
@@ -16,35 +16,47 @@ def submit
       raise Etna::NotFound.new("Workflow by the name #{session.workflow_name} could not be found.")
     end
 
-    orchestration.run_until_done!(storage)
+    begin
+      orchestration.run_until_done!(storage)
+    rescue Vulcan::Orchestration::RunErrors => e
+      run_errors = e
```

Comment:
Nice.  Yeah, admittingly this does feel odd in some sense.  One style would argue that you could always return (rather than throw an exception) a run errors, and an empty one when it happens there are no errors.  That would atleast prevent the nil vs non nil case.  But one could also argue that throwing an exception ensures that code that is written without explicit error handling is forced to see the failure.  I said use exceptions before so this is criticism of my own thinking lol.  Anyways, I don't care too much either way, just a style note.